src/main/java/frc/robot/commands/PlaceCoralAndTakeAlgae.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.*;
+
+public class PlaceCoralAndTakeAlgae extends SequentialCommandGroup {
+
+    private static final double DRIVE_BACKWARDS_SPEED = -0.2;
+    private static final double DRIVE_TIMEOUT = 0.5;
+    private static final double GRIPPER_TIMEOUT = 0.5;
+
+    public PlaceCoralAndTakeAlgae(Elevator elevator, AlgaeJoint algaeJoint, Gripper gripper,
+                                  Drivetrain drivetrain, CoralJoint coralJoint, Storage storage,
+                                  Elevator.ElevatorLevel level) {
+        addCommands(new ParallelCommandGroup(
+                        new MoveToHeight(elevator, level),
+                        new RotateStorage(coralJoint,
+                                CoralJoint.StoragePose.PLACEMENT), new RotateAlgaeJointToBottom(algaeJoint)),
```

Comment:
that was not the line break i was referring to, the commands should be on different lines for clarity. i was referring to the parameters of `RotateStorage`